modules/ui/rapid_view_manage_datasets.js

```diff
@@ -154,17 +123,70 @@ export function uiRapidViewManageDatasets(context, parentModal) {
       .attr('target', '_blank');
 
 
-    /* Pages section */
-    let pagesSection = selection.selectAll('.rapid-view-manage-pages')
-      .data([0]);
+    /* Filter section */
+    let filterEnter = selection.selectAll('.rapid-view-manage-filter')
+      .data([0])
+      .enter()
+      .append('div')
+      .attr('class', 'modal-section rapid-view-manage-filter');
 
-    let pagesSectionEnter = pagesSection.enter()
+
+    let filterSearchEnter = filterEnter
       .append('div')
-      .attr('class', 'modal-section rapid-view-manage-pages');
+      .attr('class', 'rapid-view-manage-filter-search-wrap');
+
+    filterSearchEnter
+      .call(svgIcon('#fas-filter', 'inline'));
 
-    pagesSection = pagesSection
-      .merge(pagesSectionEnter)
-      .call(renderPages);
+    filterSearchEnter
+      .append('input')
+      .attr('class', 'rapid-view-manage-filter-search')
+      .attr('placeholder', t('rapid_feature_toggle.esri.filter_datasets'))
+      .call(utilNoAuto)
+      .on('input', d3_event => {
+        const target = d3_event.target;
+        const val = (target && target.value) || '';
+        _filterText = val.trim().toLowerCase();
+        dsSection.call(renderDatasets);
+      });
+
+
+    let filterTypeEnter = filterEnter
+      .append('div')
+      .attr('class', 'rapid-view-manage-filter-type-wrap');
+
+    filterTypeEnter
```

Comment:
I noticed that the filterType combo box lets users enter whatever string they want, but I am not clear on whether we want that. It seems like we should just pre-fill the box with the known types that we infer from the datasets, and not let the user free-edit the text in it. 

Here's a screenie I took of a user typing in 'roads' thinking that they are, in fact, quite clever. The form does an excellent job letting them know they asked for something that doesn't exist, but I'm wondering if we even need to  give them that freedom. 

![image](https://user-images.githubusercontent.com/1887955/118037138-73ab9880-b33b-11eb-997f-02e28a747823.png)

Reply:
> It seems like we should just pre-fill the box with the known types that we infer from the datasets, and not let the user free-edit the text in it.

I agree - I think I can switch the combo to work that way..